### DISCUSSION TITLE
How to use colors like `success` / `info` etc as a standard bg color

### DISCUSSION THREAD

**Discussion by tommerty** [saadeghi/daisyui#2325] [Q&A] [answered]
See here for example:

```
<h1 class="bg-info">Hello</h1>
```

wouldn't work. I'm trying to build custom themes and right now just having primary/secondary isn't really enough and I'd love to be able to call success/info/accent/neutral etc as standard colors from the theme but I can't figure out how

**Comment by saadeghi** [comment, marked as answer]
It works: https://play.tailwindcss.com/4gcTGOORka  
All daisyUI color names work just like Tailwind CSS color names.  
If it's not working, I think there is an issue with your setup.
